CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to segbo
+
+
+## Prerequisites
+
+Clone the repos
+```
+git clone --recurse-submodules https://github.com/cldf-datasets/segbo
+```
+
+Install the python dependencies:
+```
+cd segbo
+pip install -e .[test]
+```
+
+## Curation workflow
+
+Update the raw data:
+```
+cldfbench download cldfbench_segbo.py
+```
+
+Recreate the CLDF dataset:
+```
+cldfbench makecldf cldfbench_segbo.py
+```
+
+Check validity of the CLDF data:
+```
+pytest
```

Comment:
@J22Melody currently fails, because a CLDF StructureDataset needs a `ValueTable`, which is not currently written. https://github.com/segbo-db/segbo/blob/master/data/SegBo%20database%20-%20Phonemes.csv would go here - I guess.